### ISSUE TITLE
ICS: missing file sRGB.icc

### ISSUE THREAD

**Issue by henning-gerhardt** [goobi/goobi-production#291]
While investigation issue #289 and after applying dirty solution mentioned in #290 next error message appear:

```
ERROR 2014-12-19 09:05:26,685 (GetMetsPdfAction.java:setPdfManagerDefaults:446)
        No ICC profile given!
```

dirty solution was (similar to #290):
- unzip ics-1.1.5.jar to temp directory
- copying  sRGB.icc to WEB-INF/classes

**Comment by frank-ulrich-weber** [comment]
The missing file sRGB.icc prevents the integrated content server to create PDF/A conform PDF files.
Default Setting in contentServerConfig.xml 
<defaultPdfConfig ... writeAsPdfA="true" .../>
The PDF file will be created anyway but not PDF/A conform. No information to the user about that fact beside the "No ICC profile given!" entry within the log file.

**Comment by henning-gerhardt** [comment]
@frank-ulrich-weber : Does it help you if you unzip `ics-1.1.5.jar` to a temporary directory and copy `sRGB.icc` to `<goobi-installation>/WEB-INF/classes` directory?

**Comment by frank-ulrich-weber** [comment]
Yes, that solves the problem.

**Comment by henning-gerhardt** [comment]
Then should both files `sRGB.icc` and `errorfile.jpg` (from #290) should be added to `WEB-INF/classes` directory.

**Comment by henning-gerhardt** [comment]
Fixed with pull request #418 (https://github.com/goobi/goobi-production/commit/1688ffc588492bb23a35d5612c0c5a3be64588c8)
